rules/integrations/github/persistence_github_new_user_added_to_organization.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2023/10/11"
+integration = ["github"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/12/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+A new user was added to a GitHub organization.
+"""
+from = "now-9m"
+index = ["logs-github.audit-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "New User Added To GitHub Organization"
+risk_score = 21
+rule_id = "61336fe6-c043-4743-ab6e-41292f439603"
+severity = "low"
+tags = ["Domain: Cloud",
```

Comment:
Similar to remove member, should we make this a BBR instead? A SOC analyst, for instance, may be inclined to consider this not a threat without more context.

Reply:
Good point, I'll move this to BBR as well